### DISCUSSION TITLE
Feature Request: Skip back or ahead buttons.

### DISCUSSION THREAD

**Discussion by vbmark** [YouTube-Enhancer/extension#485] [Ideas]
It would be handy if I could display two buttons "Below Player".  One that skips backwards 5 seconds and one that skips forwards 5 seconds.

Thank you very much!

**Comment by VampireChicken12** [comment]
@vbmark thanks for the suggestion, do you mind making a feature request issue with the same info. So I can put the issue into the kanban board.

**Reply by vbmark** [reply]
No problem. I had to Google how to do that, but it's there now.  Thank you.

**Reply by VampireChicken12** [reply]
> No problem. I had to Google how to do that, but it's there now.  Thank you.

Thank you
